lib/ae_mdw/activities.ex

```diff
@@ -527,21 +520,20 @@ defmodule AeMdw.Activities do
     }
   end
 
-  defp render(state, _account_pk, height, txi, {:aexn, :aex141, from_pk, to_pk, value, index}) do
+  defp render(state, _account_pk, height, txi, {:aexn, :aex9, from_pk, to_pk, value, index}) do
     %{
       height: height,
-      type: "Aex141TransferEvent",
-      payload: render_aexn_payload(state, txi, from_pk, to_pk, value, index)
+      type: "Aex9TransferEvent",
+      payload: AexnView.sender_transfer_to_map(state, {:aex9, from_pk, txi, to_pk, value, index})
```

Comment:
I've renamed these attributes to match txs format. I also left adding `key_block_hash` at the top level because I found that it requires a lot more changes. Leaving that up for the next PR